_includes/code/quickstart/endtoend.py

```diff
@@ -95,13 +96,18 @@
         query="biology",
         limit=2
     )
-
-    print(response.objects[0].properties)  # Inspect the first object
+    
+    # Extract the properties of the first two objects
```

Comment:
I think we should leave this as is. Additional info can be helpful. The downside is that navigating the additional printing code makes ti more difficult to focus on the query code.